tensorflow_probability/python/mcmc/sample_test.py

```diff
@@ -56,6 +58,29 @@ def is_calibrated(self):
     return self._is_calibrated
 
 
+class RandomTransitionKernel(tfp.mcmc.TransitionKernel):
+  """Fake `TransitionKernel` that randomly assigns the next state.
+
+  Regardless of the current state, the `one_step` method will always
+  randomly sample from a Reyleigh Distribution.
+  """
+
+  def __init__(self, is_calibrated=True, accepts_seed=True):
+    self._is_calibrated = is_calibrated
+    self._accepts_seed = accepts_seed
+
+  def one_step(self, current_state, previous_kernel_results, seed=None):
+    if seed is not None and not self._accepts_seed:
+      raise TypeError('seed arg not accepted')
+    random_next_state = tfp.random.rayleigh((), seed=seed)
```

Comment:
I know this is just a test, but I'd recommend doing `tfp.random.rayleigh(current_state.shape, seed=seed)`, so that this kernel supports multiple chains.

Reply:
Good point!